reads/start.sh

```diff
@@ -7,4 +7,4 @@ export NODE_ENV="development"
 DEFAULT_PORT="8000"
 export PORT=${PORT:-$DEFAULT_PORT}
 
-yarn run nodemon ./src/server.js
+pnpm nodemon ./src/server.js
```

Comment:
The reads api calls `node src/server.js` (as well as `yarn install`) in deploy/dockerfiles/reads/reads-api.dockerfile to startup. Do we need to update the build in that dockerfiles/reads service as well? or will updating the reads service be a subsequent PR?

Reply:
Noting that the variant page crashes on the current demo.  Could be related to this?